src/xbps/packages/downgrading.md

```diff
@@ -0,0 +1,28 @@
+# Downgrading to a specific package version
+
+## Via xdowngrade
+
+The easiest way to downgrade is to use `xdowngrade` from the `xtools` package,
+specifying the package version you wish to downgrade to:
+
+```
+# xdowngrade /var/cache/xbps/pkg-1.0_1.xbps
+```
+
+## Via XBPS
+
+To downgrade using just XBPS, you must first add the desired package version to
```

Comment:
To be clear, this is adding a locally cached older version of the package which is no longer contained in the repository index, correct? (or could be used if you got an older version somewhere else too.)

Reply:
Thanks for the feedback - PR updated.